freqtrade/optimize/lookahead_analysis_helpers.py

```diff
@@ -119,25 +119,22 @@ def add_or_update_row(df, row_data):
         csv_df.to_csv(config['lookahead_analysis_exportfilename'], index=False)
 
     @staticmethod
-    def initialize_single_lookahead_analysis(strategy_obj: Dict[str, Any], config: Dict[str, Any]):
-
-        logger.info(f"Bias test of {Path(strategy_obj['location']).name} started.")
-        start = time.perf_counter()
-        current_instance = LookaheadAnalysis(config, strategy_obj)
-        current_instance.start()
-        elapsed = time.perf_counter() - start
-        logger.info(f"checking look ahead bias via backtests "
-                    f"of {Path(strategy_obj['location']).name} "
-                    f"took {elapsed:.0f} seconds.")
-        return current_instance
-
-    @staticmethod
-    def start(config: Config):
+    def calculate_config_overrides(config: Config):
```

Comment:
You might wanna add an explicit test for this.
It's nice and isolated, which makes this a pretty easy test to write - but we'll want to ensure this remains correct for the future - as it's a critical piece of functionality to have the bias checker work correctly.

Reply:
true, sure